packages/lit-dev-tools-cjs/src/api-docs/transformer.ts

```diff
@@ -480,7 +498,14 @@ export class ApiDocsTransformer {
           key === 'tags' ||
           // The "target" key is unstable causing our "Check API data is in
           // sync" approach to fail. We also do not use this key.
-          key === 'target'
+          key === 'target' ||
```

Comment:
We're now using the key of 'target' for referencing reflected node as per the PR description, and usage in line 425. Do we now want to keep this around?

Reply:
Correct that we use `target` and `id` for tracking declarations and references to declarations. This just filters out keys that we don't need for `pages.json` -> the rendered html. So we can remove `target` from the cached output.